builders/integrations/indexers/covalent.md

```diff
@@ -46,70 +56,356 @@ If you're already familiar with Covalent and ready to dive in, you simply need t
+ - Note that free development API keys are rate limited to `4` requests per second. Subscribers to a professional plan can make up to `50` requests per second.
  - The root URL of the API is: `https://api.covalenthq.com/v1/`
  - All requests are done over HTTPS (calls over plain HTTP will fail)
- - The refresh rate of the APIs is real-time: 30s or 2 blocks, and batch 30m or 40 blocks
+ - The refresh rate of the APIs is real-time: 30s or two blocks, and batch 30m or 40 blocks
 
 ### Types of Endpoints {: #types-of-endpoints }
 
-The Covalent API has three classes of endpoints:
+The GoldRush API has three classes of endpoints:
 
  - **Class A** — endpoints that return enriched blockchain data applicable to all blockchain networks, eg: balances, transactions, log events, etc
  - **Class B** — endpoints that are for a specific protocol on a blockchain, e.g. Uniswap is Ethereum-only and is not applicable to other blockchain networks
- - **Class C** — endpoints that are community built and maintained but powered by Covalent infrastructure
+ - **Class C** — endpoints that are community-built and maintained but powered by GoldRush infrastructure
 
 ### Sample Supported Endpoints {: #sample-supported-endpoints }
 
-For a full list of supported endpoints, refer to the [Covalent API reference](https://www.covalenthq.com/docs/api/guide/overview){target=\_blank}. A subset of the supported endpoints include:
+For a full list of supported endpoints, refer to the [GoldRush API reference](https://goldrush.mintlify.app/api-reference/overview){target=\_blank}. A subset of the supported endpoints include:
 
- - **Balances** — get token balances for an address. Returns a list of all ERC-20 and NFT token balances including ERC-721 and ERC-1155 along with their current spot prices (if available)
- - **Transactions** — retrieves all transactions for an address including decoded log events. Does a deep-crawl of the blockchain to
- retrieve all transactions that reference this address
- - **Transfers** — get ERC-20 token transfers for an address along with historical token prices (if available)
- - **Token holders** — return a paginated list of token holders
- - **Log events (smart contract)** — return a paginated list of decoded log events emitted by a particular smart contract
- - **Log events (topic hash)** — return a paginated list of decoded log events with one or more topic hashes separated by a comma
- - **Security (Token Approvals)** - return a list of approvals across all token contracts categorized by spenders for a wallet’s assets
+- **Token Balances**: Get all token balances (native, ERC20, ERC721, ERC1155) with current market prices for an address
```

Comment:
These list items should follow these guidelines:

- Only the first word needs to be capitalized, unless it's something specific like a product name or the norm to capitalize it
- Use a dash instead of a colon between the title and the description
- The first word of the description doesn't need to be capitalized

So for example, this line should be like `- **Token balances** - get all...`